src/patternfly/components/Select/select.scss

```diff
@@ -395,18 +359,17 @@
   color: var(--pf-c-select__menu-item--Color);
   text-align: left;
   white-space: nowrap;
-  background-color: transparent; // this can be removed if the shield in _base.css works for FF
```

Comment:
This is needed now that we removed `* {background-color: transparent;}`. Let's chat about it tomorrow.